artiq/coredevice/comm_mgmt.py

```diff
@@ -63,6 +63,13 @@ def open(self):
         self.socket = socket.create_connection((self.host, self.port))
         logger.debug("connected to %s:%d", self.host, self.port)
         self.socket.sendall(b"ARTIQ management\n")
+        endian = self._read(1)
+        if endian == b"e":
+            self.endian = "<"
+        elif endian == b"E":
+            self.endian = ">"
+        else:
+            raise IOError("Incorrect reply from device: expected e/E.")
```

Comment:
Nitpicking, but it's using ``ValueError`` elsewhere.

Reply:
`ValueError` is used for analyzer as the error could occur when reading dumped analyzer output, which is not an IO error. Otherwise, it should be an IO error.